website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -0,0 +1,346 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_schedule"
+sidebar_current: "docs-mongodbatlas-resource-cloud-backup-schedule"
+description: |-
+    Provides a Cloud Backup Schedule resource.
+---
+
+# mongodbatlas_cloud_backup_schedule
+
+`mongodbatlas_cloud_backup_schedule` provides a cloud backup schedule resource. The resource lets you create, read, update and delete policies items.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
```

Comment:
In the examples below the use of the word step is going to be confusing as that implies you need to do an action, and then another.  It will make people think this takes two applies.  I'll make some suggestions to clear them up hopefully.